client/signup/steps/design-type-with-store/index.jsx

```diff
@@ -158,7 +156,11 @@ class DesignTypeWithStoreStep extends Component {
 		return (
 			<div className="design-type-with-store__substep-wrapper">
 				<div className={ storeWrapperClassName }>
-					{ this.renderStoreStep() }
+					<PressableStoreStep
+						{ ... this.props }
+						onBackClick={ this.handleStoreBackClick }
+						setRef={ this.setPressableStore }
+					/>;
```

Comment:
Ooops! :D Thanks for catching this @bisko! Removed it in b09f941aa59c851a91a917cad86530d138879a45.